bin/savepdf

```diff
@@ -35,5 +37,6 @@ if (program.preview) {
   preview(path.resolve(process.cwd(), program.args[0]), program.sandbox);
```

Comment:
In preview mode, it will be no affection about the `--root` option. Could you fix it?